README.md

```diff
@@ -309,6 +309,21 @@ git = "https://github.com/japaric/steed"
 stage = 2
 ```
 
+## Cargo mode
+
+Xaro supports configuring the mode in which `cargo` is invoked.
```

Comment:
Specifically this affects only the sysroot cargo, right? Typical use of xargo involves also having an actual project that is then built once libstd is done.